### PR TITLE
Init/typescript

### PR DESCRIPTION
**Pull request by shamoons**

### CODE REVIEW

**Comment by shamoons** [comment]
@kfaria - any bad code I should fix?

File: src/index.peer.ts.bak [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  })
+
+  req.end()
+
+  console.log('Listening on port ' + PORT)
+  return server.listen(PORT)
+}
+
+bootstrap()
+
```
Comment (kfaria) [inline src/index.peer.ts.bak:53]: Is there a need for a backup file?

File: src/index.ts [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  .description('A node in the network')
+  .option('-p, --port <port>', 'Port to listen to')
+
+program.parse();
+
+const options = program.opts()
+const PORT = options.port as number || 10309
+
+const requestHandler = (request: http.IncomingMessage, response: http.ServerResponse) => {
+  console.log('I GOT SOMETHING!!!!')
```
Comment (kfaria) [inline src/index.ts:15]: Lets remove this console.log -> can replace it with something along the lines of "request received"

**Review by kfaria — APPROVED** [review]
small fixes then we can merge in!  @shamoons

File: src/index.ts (hunk 2 above)
Comment (shamoons) [reply src/index.ts]: Removed

**Pull request merged**